<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EbDialogGroupInfo</class>
 <widget class="QDialog" name="EbDialogGroupInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelGroupName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>group name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGroupName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelManager">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>tel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>description</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFax">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>fax</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUrl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFax">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUrl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>260</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisplayIndex">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>display index</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDisplayIndex">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditDescription">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
